include/tvm/runtime/c_runtime_api.h

```diff
@@ -207,6 +209,7 @@ typedef DLTensor* TVMArrayHandle;
  */
 typedef union {
   int64_t v_int64;
+  bool v_bool;
```

Comment:
likely v_bool is not needed and we can simply store value in `v_int64 `, this can help to reduce the switch cases, and also make all value types as 64 bit, might help kernel passing handling to reduce cases as well.